src/Tmds.Systemd.Logging/JournalLoggerOptions.cs

```diff
@@ -9,5 +11,9 @@ public class JournalLoggerOptions
         public bool DropWhenBusy { get; set; }
         /// <summary>The syslog identifier added to each log message.</summary>
         public string SyslogIdentifier { get; set; } = Journal.SyslogIdentifier;
+        /// <summary>Sets the full exception field instead of separate fields.</summary>
+        public bool SetFullException { get; set; }
+        /// <summary>Formats a full exception. If unset, <see cref="Exception.ToString"/> is used.</summary>
+        public Func<Exception, string> FullExceptionFormatter { get; set; }
```

Comment:
Let's drop the `SetFullException`, and change `FullExceptionFormatter` to `Func<Exception, JournalMessage> ExceptionFormatter`

When the `ExceptionFormatter` is not null, we call it, and it can add any fields it likes. For example:
```
ExceptionFormatter = (ex, msg) => msg.Append("EXCEPTION", ex.ToString());
```

Reply:
Yes, that would also be okay. I was also hoping for some kind of "standardisation" or at least recommendation so that not every application does it differently. A common approach would make it easier to find and display the exceptions in log viewers, just like you have already set names for the existing exception fields.

Reply:
We can expose the default behavior as a static property. That allows the user to call into it from his custom handler, and add additional fields.

Reply:
Sounds good, I'll take a look at it.

Reply:
Oh, an upcoming release is a good motivation. :-) I'm a bit uncertain about the field names. I could leave it as FULL_EXCEPTION, but I might also use EXCEPTION for the full string and leave out the other fields. The difference could be determined when EXCEPTION exists but EXCEPTION_TYPE not. Whatever the field name, it needs to be provided by the `JournalLoggerOptions` class if we want anybody to consider it. I'll just push another commit with the changes so far.